projects/odf_transform/odf_transform/odf_parser/odf_parser.py

```diff
@@ -39,32 +30,24 @@ def read(filename,
             d. Each section items are grouped as a dictionary
             e. dictionary items are converted to datetime (deactivated), string, integer or float format.
         2. Read the data  following the header with Pandas.read_csv() method
-            a. Use defined separator  to distinguish columns (default '\s+').
+            a. Use defined separator  to distinguish columns (default "\s+").
             b. Convert each column of the pandas data frame to the matching format specified in
             the TYPE attribute of the ODF associated PARAMETER_HEADER
 
     read_odf is a simple tool that  parse the header metadata and data from an DFO ODF file to a list of dictionaries.
     :param filename: ODF file to read
-    :param header_end: Expression used at the end of a ODF file header to define the end of the header and
+    :param encoding_format: odf encoding format
      start of the data.
-    :param data_delimiter: delimiter used by the odf separate the different data columns.
-    :param quotechar: quote character to regroup string variables
-    :param parameter_section: section of the ODF that describes each variable attributes
-    :param output_column_name: variable attribute used to name each data columns
-    :param variable_type: variable attribute that describe each variable type
-    :param section_items_minimum_whitespaces: maximum amount of spaces prior to a section to be considered as a section
-    :param odf_type_to_pandas: dictionary which map odf types versus python(Pandas) types.
     :return:
     """
-    # Default mapping of ODF to Pandas data type
-    if odf_type_to_pandas is None:
-        odf_type_to_pandas = odf_dtypes
-
-    odf_date_format = {'SYTM': {'regex': r'^\s*\'\d\d\-\w\w\w\-\d\d\d\d\s\d\d\:\d\d\:\d\d\.\d*\'\s*$',
-                                'datetime': '%d-%b-%Y %H:%M:%S.%f'},
-                       'header': {'regex': r'^\'\d\d\-\w\w\w\-\d\d\d\d\s\d\d\:\d\d\:\d\d\.\d*\'$',
-                                  'datetime': '\'%d-%b-%Y %H:%M:%S.%f\''}
-                       }
+    header_end = '-- DATA --'
+    data_delimiter = r'\s+'
+    quotechar = '\''
+    parameter_section = 'PARAMETER_HEADER'
+    variable_type = 'TYPE'
+    null_value = 'NULL_VALUE'
+    section_items_minimum_whitespaces = 2
+    original_prefix_var_attribute = 'original_'
 
     def _convert_to_number(value):
```

Comment:
not part of this PR but looks like this function has some issues, eg 
`_convert_to_number(5.4)` = 5

Reply:
oh yeah very good point! Float and then int!

Reply:
This was corrected!

Reply:
that was corrected but should have been actually need to be corrected in next PR